internal/proto/controller/api/services/v1/account_service.proto

```diff
@@ -13,68 +13,153 @@ import "protoc-gen-openapiv2/options/annotations.proto";
 
 option go_package = "github.com/hashicorp/boundary/internal/gen/controller/api/services;services";
 option (custom_options.v1.domain) = "auth";
+// Global definitions for our API, have to go in one of the protobuf files.
+// Putting it in the file that's alphabetically sorted first means it maintains
+// the alphabetic ordering of services.
```

Comment:
Hm, it could certainly go in a separate file 👍🏻. The comment is incomplete, what I noticed was that adding it to the scope service file seemed to sort that one to the top, so I moved it to this file, but a separate file is a better idea 👍🏻.

Reply:
Moved to a separate file